arrow-libs/fx/arrow-fx-coroutines/src/commonMain/kotlin/arrow/fx/coroutines/ParZip.kt

```diff
@@ -41,7 +45,14 @@ public suspend inline fun <A, B, C> parZip(
   crossinline fa: suspend CoroutineScope.() -> A,
   crossinline fb: suspend CoroutineScope.() -> B,
   crossinline f: suspend CoroutineScope.(A, B) -> C
-): C = parZip(Dispatchers.Default, fa, fb, f)
+): C {
+  contract {
+    callsInPlace(fa, InvocationKind.AT_MOST_ONCE)
+    callsInPlace(fb, InvocationKind.AT_MOST_ONCE)
+    callsInPlace(f, InvocationKind.EXACTLY_ONCE)
```

Comment:
Shouldn't this one be also `AT_MOST_ONCE`? If we continue with the reasoning that makes `fa` and `fb` be called `AT_MOST_ONCE`, I think those should be there too.